forumdirectory/forumdirectory.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Name: Forum Directory
  * Description: Add a directory of forums hosted on your server, with verbose descriptions.
- * Version: 1.0
+ * Version: 1.1
  * Author: Thomas Willingham <https://beardyunixer.com/profile/beardyunixer>
  */
```

Comment:
You can add yourself to the list of authors if you want.

Reply:
I don't want to :-)